lib/utils/computed-properties.js

```diff
@@ -38,32 +38,38 @@ function isComputedPropertyBodyArg(node) {
  * @returns {ASTNode} function body of computed property
```

Comment:
Can you update the jsdoc comment with additional examples (`@computed()` and `@computed`), and mention the parameter can be a CallExpression or Identifier?

Reply:
I updated the doc for `getComputedPropertyFunctionBody`, but not this one, because it seems to be only intended for when you're passing something in as the last argument to `computed`.